atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/Transaction.java

```diff
@@ -41,25 +41,95 @@
  */
 public interface Transaction {
 
+    /**
+     * Returns a mapping of rows to {@link RowResult}s within {@code tableRef} for the specified {@code rows}, loading
+     * columns according to the provided {@link ColumnSelection}.
+     *
+     * The returned {@link SortedMap} is sorted on the byte order of row keys; the ordering of the input parameter
+     * {@code rows} is irrelevant. Access to the returned {@link SortedMap} is permitted both with the original byte
```

Comment:
'with any byte array that is equivalent according to Arrays.equals(byte[], byte[])'.

Reply:
Yep, of course the same array is equal to itself!